samples/java-valueentity-counter-spring-client/pom.xml

```diff
@@ -52,7 +52,7 @@
         <dependency>
             <groupId>net.devh</groupId>
             <artifactId>grpc-client-spring-boot-starter</artifactId>
-            <version>2.13.1.RELEASE</version>
+            <version>2.14.0.RELEASE</version>
```

Comment:
This is another third party library to have grpc client. I'm just updating to last version. 

Their README mention it is build with 2.6.13, https://github.com/yidongnan/grpc-spring-boot-starter#versions not clear if it will work with SpringBoot 3

Reply:
SpringBoot 3 uses Spring Framework 6 and may not be compatible with the grpc starter.

https://github.com/yidongnan/grpc-spring-boot-starter/issues/826

If that's the case, version 2.13.1 won't be a good fit either and we will have to keep this sample on SpringBoot 2.7.8

Reply:
Ok, tests failing with 

```java
java.lang.NoClassDefFoundError: javax/servlet/DispatcherType
	at com.example.client.spring.rest.CounterControllerIntegrationTest.setUp(CounterControllerIntegrationTest.java:27)
Caused by: java.lang.ClassNotFoundException: javax.servlet.DispatcherType
	at com.example.client.spring.rest.CounterControllerIntegrationTest.setUp(CounterControllerIntegrationTest.java:27
```

We won't be able to update this sample for now. Unless grpc-spring-boot-starter gets updated or we have another solution for it. 

I don't think it's a bit issue for this sample to stay on Boot 2.7.8.

Reply:
https://app.circleci.com/pipelines/github/lightbend/kalix-jvm-sdk/5467/workflows/d24e1efa-1af3-4e31-91c4-ddd7b9e64593/jobs/103012?invite=true#step-113-446

```
java.lang.NoClassDefFoundError: Could not initialize class com.github.tomakehurst.wiremock.jetty9.JettyHttpServerFactory
	at com.example.client.spring.rest.CounterControllerIntegrationTest.setUp(CounterControllerIntegrationTest.java:27)
```